Usage Docs link added to README and Docs Front page

canyon289 opened the pull request:
Will merge after checks are done. I can't imagine these changes will be very controversial and I'd like to send Stefan an email by tonight.

ColCarroll commented on README.md:
```diff
@@ -13,6 +13,14 @@ Includes functions for posterior analysis, model checking, comparison and diagno
 The official ArviZ documentation can be found here
 https://arviz-devs.github.io/arviz/index.html
```
Not your change, not entirely necessary, but can we change this to an inline link, and combine these three short sentences into one paragraph (they'll need periods at the end of each)? Here's the link I mean:

The official ArviZ documentation can be found [here](https://arviz-devs.github.io/arviz/index.html).

ColCarroll commented on README.md:
```diff
@@ -10,8 +10,10 @@ Includes functions for posterior analysis, model checking, comparison and diagno
 
-The official ArviZ documentation can be found here
-https://arviz-devs.github.io/arviz/index.html
+The ArviZ documentation can be found in the [official docs](https://arviz-devs.github.io/arviz/index.html) First time users may find the [quickstart](https://arviz-devs.github.io/arviz/notebooks/Introduction.html).
```
```suggestion
The ArviZ documentation can be found in the [official docs](https://arviz-devs.github.io/arviz/index.html). First time users may find the [quickstart](https://arviz-devs.github.io/arviz/notebooks/Introduction.html)
```

ColCarroll commented on README.md:
```diff
@@ -10,8 +10,10 @@ Includes functions for posterior analysis, model checking, comparison and diagno
-https://arviz-devs.github.io/arviz/index.html
+The ArviZ documentation can be found in the [official docs](https://arviz-devs.github.io/arviz/index.html) First time users may find the [quickstart](https://arviz-devs.github.io/arviz/notebooks/Introduction.html).
+to be helpful. Additional guidance can be found in the
+[usage documentation](https://arviz-devs.github.io/arviz/usage.html)
```
```suggestion
[usage documentation](https://arviz-devs.github.io/arviz/usage.html).
```

canyon289 replied:
Ugh getting sloppy. Thank you. After this I'm done for tonight :)

ColCarroll replied:
:+1: i can merge in a few minutes -- thanks much!

canyon289 commented:
Incorporated feedback. I also changed the link text to be more descriptive than `here` because the browser turns the link text blue and makes it stand out.
![arvizreadme](https://user-images.githubusercontent.com/7213793/48595714-c5aa0980-e90a-11e8-8c0e-55e4762fc444.png)
